src/components/DataDuesAction/fields.js

```diff
@@ -1,8 +1,91 @@
 import React, { useEffect } from 'react'
 import PropTypes from 'prop-types'
 import NumberFormat from 'react-number-format'
+import AlgoliaPlaces from 'algolia-places-react'
+import classNames from 'classnames'
 import { Form } from 'react-bootstrap'
 
+export const AlgoliaPlacesField = ({
+  name,
+  register,
+  unregister,
+  setValue,
+  defaultValue,
+  isInvalid,
+  ...props
+}) => {
+  useEffect(() => {
+    register(name)
+    return () => unregister(name)
+  }, [name, register, unregister])
+
+  const fieldName = name
+  const cssClasses = classNames('form-control', { 'is-invalid': isInvalid })
+
+  console.log('classes', cssClasses)
```

Comment:
I will expect our linting should avoid this... Odd that doesn't happen.
_(haven't re-check the config we have tho)_

Reply:
ok let me remove this